src/classes/REL_Relationships_TDTM.cls

```diff
@@ -218,8 +210,7 @@ public with sharing class REL_Relationships_TDTM extends TDTM_Runnable {
         for(npe4__Relationship__c r : Relationships) {
         	
             //SWA 2009-05-13 removed a duplicate check for change to npe4__Status__c from this OR statement
-            if ((r.npe4__Status__c != oldRelationships[counter].npe4__Status__c 
-            || r.npe4__Type__c != oldRelationships[counter].npe4__Type__c 
+            if ((r.npe4__Type__c != oldRelationships[counter].npe4__Type__c 
```

Comment:
what if we only updated type on the reciprocal rel, if type was changed on the first rel.  ie, if the rel being updated hasn't changed its type, then we avoid calling the type routine.